packages/builder/src/pages/builder/app/[application]/settings/backups/index.svelte

```diff
@@ -221,28 +256,42 @@
               bind:value={filterOpt}
             />
           </div>
-          <DateRangePicker
-            value={dateRange}
-            on:change={e => (dateRange = e.detail)}
-          />
+          <div class="date-range-compact">
+            <DateRangePicker
+              value={dateRange}
+              on:change={e => (dateRange = e.detail)}
+            />
+          </div>
         </div>
-        <div>
+        <div class="actions">
+          {#if hasSelection}
+            <div class="selection-controls">
+              <Button
+                warning
+                on:click={bulkDeleteDialog.show}
+                disabled={!hasSelection}
```

Comment:
This doesn't make sense because `hasSelection` is true here, so it will never be disabled.